pkgs/top-level/all-packages.nix

```diff
@@ -24181,4 +24181,6 @@ in
 
   dapper = callPackage ../development/tools/dapper { };
 
+  pixiecore = callPackage ../tools/networking/pixiecore {}; 
```

Comment:
This is not the right place, it should go under the `### TOOLS` heading (a good trick is to grep for `tools/networking` and place it close to a match)